pilot/pkg/model/config.go

```diff
@@ -300,6 +300,10 @@ const (
 
 	// NamespaceAll is a designated symbol for listing across all namespaces
 	NamespaceAll = ""
+
+	configGroup = "config"
```

Comment:
do not define constants here. it doesn't help to clarify the model. Please write "networking" or "config" in each proto schema. 

i plan to remove istioAPIVersion once the everything is done.